### PR TITLE
Fixed co example

### PR DESCRIPTION
**Pull request by odino**
;-)

### CODE REVIEW

File: examples/promise-co-await/co.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
-var mysql = require('../promise.js');
+var mysql = require('../../promise.js');
```
Comment (sushantdhiman) [inline examples/promise-co-await/co.js]: Actually change it to `require('mysql/promise')`

Comment (odino) [inline examples/promise-co-await/co.js]: then it should be changed [everywhere](https://github.com/sidorares/node-mysql2/blob/master/examples/mysqlproxy.js)?

Comment (sushantdhiman) [inline examples/promise-co-await/co.js]: Right, you can fix this example, I'll handle the rest :)

Comment (odino) [inline examples/promise-co-await/co.js]: though one thing is that the examples wont work out of the box -- ie I just cloned the repo and ran node examples/promise-co-await/co.js -- works like a charm. If we use absolute requires you wont be able to do that. Happy to change stuff as long as you think that's alright!

Comment (sushantdhiman) [inline examples/promise-co-await/co.js]: I think these examples are meant to be copied when user have installed the `node-mysql2` package. So please go ahead and commit the change as discussed

**Comment by sushantdhiman** [comment]
@sidorares What do you think about making all `examples` standalone, right now they cant be copied and tested. I think most developers would like to quickly copy the code and do `npm install mysql2`, we should make all of these examples standalone

**Comment by sidorares** [comment]
@sushantdhiman sorry forgot about this
shouldn't examples `require('mysql2/promise')` instead of mysql/promise ?

**Comment by sushantdhiman** [comment]
I realized that and fixed all case I found in #429

**Comment by sidorares** [comment]
yeah, saw that after commented, 👍

**Pull request merged**